tracks/golang/_posts/2024-01-05-golang-week-2.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Week 2 of Go Lang
+feature_image: "https://picsum.photos/2560/600?image=872"
+excerpt: |
+  ### Week two course content
+feature_text: |
+  ## GoLang Week 2
+  
+
+---
+
+
+#### Variables in golang
+[Resourse Link](https://youtu.be/pM0-CMysa_M?feature=shared)
```

Comment:
For youtube links - you could add them into an iframe usin this include tag - `{% include video.html id="4rLW7zg21gI" %}` where 4rLW7zg21gI is the youtube video id.